rules.html

```diff
@@ -0,0 +1,171 @@
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta
+			name="keywords"
+			content="otc, talks, OTC Talks, Our Tech Community"
+		/>
+		<meta name="description" content="Rules for OTC Talks." />
+		<meta name="author" content="Harsh Kapadia" />
+		<meta property="og:site_name" content="OTC Talks" />
+		<meta property="og:title" content="Rules" />
+		<meta property="og:type" content="website" />
+		<meta property="og:description" content="Rules for OTC Talks." />
+		<meta
+			property="og:url"
+			content="https://talks.ourtech.community/rules"
+		/>
+		<meta
+			property="og:image"
+			content="https://talks.community/static/img/og-img/home.png"
+		/>
+		<meta property="og:image:width" content="1200" />
+		<meta property="og:image:height" content="600" />
+		<meta property="twitter:card" content="summary_large_image" />
+		<meta
+			property="twitter:image"
+			content="https://ourtech.community/static/img/og-img/home.png"
+		/>
+
+		<script defer src="static/js/script.js"></script>
+
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+			rel="stylesheet"
+		/>
+
+		<link rel="stylesheet" href="static/css/style.css" />
+
+		<link
+			rel="shortcut icon"
+			href="static/img/favicon.ico"
+			type="image/x-icon"
+		/>
+
+		<title>OTC Talks Rules</title>
+	</head>
+
+	<body>
+		<header>
+			<nav>
+				<div class="burger-menu">
+					<div class="line-1"></div>
+					<div class="line-2"></div>
+					<div class="line-3"></div>
+				</div>
+
+				<ul class="navbar-ul hide-burger-menu burger-menu-dropdown">
+					<li><a href="/">Home</a></li>
+					<li><a href="/register">Register</a></li>
+					<li><a href="/rules">Rules</a></li>
+					<li>
+						<a
+							href="https://ourtech.community/coc"
+							target="_blank"
+							rel="noreferrer"
+							>Code of Conduct</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://t.me/OurTechComm"
+							target="_blank"
+							rel="noreferrer"
+							>Join us on Telegram</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/two-years"
+							target="_blank"
+							rel="noreferrer"
+							>Two Years of OTC</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/team"
+							target="_blank"
+							rel="noreferrer"
+							>Our Team</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://links.ourtech.community"
+							target="_blank"
+							rel="noreferrer"
+							>All Links</a
+						>
+					</li>
+				</ul>
+			</nav>
+		</header>
+
+		<h1>OTC Talks Rules</h1>
+
+		<main id="main-content">
+			<ul>
+				<li>The mode of OTC Talks is Online.</li>
+				<li>
+					Duration of the lightning talk: 15 - 25 minutes from 10:45
+					PM IST on a Saturday, followed by the CatchUp on the

```

Comment:
`CatchUp` should be `OTC CatchUp` and should be hyperlinked to the OTC CatchUp home page. (I'm not sure whether it should be opened in a new page or on the same page itself.)

Reply:
Yep.